situp-core/src/main/java/com/amazon/situp/pipeline/PipelineConnector.java

```diff
@@ -44,14 +52,25 @@ public void output(final Collection<T> records) {
                 try {
                     buffer.write(record, DEFAULT_WRITE_TIMEOUT); //TODO update to use from config
```

Comment:
nit: You can drop the TODO, i dont think we will allow custom values because a runtimeexception is going to crash all the pipelines.